packages/odyssey-react/src/components/Form/Form.tsx

```diff
@@ -64,27 +65,39 @@ export const Form = withTheme(
     const omitProps = useOmit(rest);
 
     return (
-      <form {...omitProps} className={styles.root} ref={ref}>
-        <header className={styles.header}>
+      <Box as="form" {...omitProps} className={styles.root} ref={ref}>
+        <Box as="header" className={styles.header}>
```

Comment:
I know we're not fully decided yet but, for me this inner header doesn't need to be Box